documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -253,7 +477,41 @@ A complete connector JSON config will look like that:
     "topic.creation.applicationlogs.compression.type": "lz4"
 }
 ----
+endif::community[]
 
+ifdef::product[]
+[source,yaml]
+----
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaConnector
+metadata:
+  name: inventory-connector
+...
+spec:
+...
+
+   config:
+...
+    topic.creation.default.replication.factor: 3,
+    topic.creation.default.partitions: 10,
+    topic.creation.default.cleanup.policy: compact
+    topic.creation.default.compression.type": lz4
+    topic.creation.groups: inventory,applicationlogs
+    topic.creation.inventory.include: dbserver1\\.inventory\\.*
+    topic.creation.inventory.replication.factor: 3
```

Comment:
again, not needed to repeat the default group value here, as long there's no foreseeable reason to update the default group's value in the future and keep that value for this group.

Reply:
Thanks for your close review. Sorry about the silly mistakes. :(